mio/data/config/process/denoise_example.yml

```diff
@@ -20,6 +20,9 @@ noise_patch:
   output_diff: true
   output_noisy_frames: true
 frequency_masking:
+  id: frequency_masking_example
+  mio_model: mio.models.process.FreqencyMaskingConfig
+  mio_version: 0.6.1
   enable: true
```

Comment:
It's somewhat awkward having to do this to make the FrequencyMaskingConfig an actual configuration that can also work independently, but it's also convenient to have all configurations in a single file.

Should I make this a separate file, or could (should) we possibly change the ConfigYAMLMixin to make the ID only necessary for non-nested configs?